src/families/location_scale.jl

```diff
@@ -113,16 +113,21 @@ function Distributions._rand!(
     return x .+= location
 end
 
-Distributions.mean(q::MvLocationScale) = q.location
+function Distributions.mean(q::MvLocationScale)
+    @unpack location, scale = q
+    return location + scale * Fill(mean(q.dist), length(location))
+end
 
 function Distributions.var(q::MvLocationScale)
     C = q.scale
-    return Diagonal(C * C')
+    σ2 = var(q.dist)
+    return σ2 * diag(C * C')
 end
 
 function Distributions.cov(q::MvLocationScale)
     C = q.scale
-    return Hermitian(C * C')
+    σ2 = var(q.dist)
+    return σ2 * Hermitian(C * C')
```

Comment:
I don't know the theory here well, but is there a reason why this involves `var(q.dist)` rather than `cov(q.dist)`? I could have imagined it being something like `C * cov(q.dist) * C'`, though that's just a not-very-educated guess.

Reply:
Good point. I was thinking that `q.dist` was constrained to be a univariate distribution, which would make all of this valid, but seems like I have to use `ContinuousUnivariateDistribution` for that. Let me fix this later.

Reply:
Oh I see, yeah, this makes sense for univariate. Is there a reason you want to restrict to `q.dist` being univariate? Just less of a headache to implement?